c++/src/capnp/rpc.c++

```diff
@@ -910,15 +910,17 @@ private:
         // ultimately resolves to a local capability. Instead, we'll need to queue the call until
         // the promise resolves.
 
-        auto vpapPromises = fork.addBranch().then(kj::mvCapture(context,
-            [interfaceId,methodId](kj::Own<CallContextHook>&& context,
-                                   kj::Own<ClientHook> resolvedCap) {
+        kj::Tuple<kj::Promise<void>, kj::Promise<kj::Own<PipelineHook>>> vpapPromises =
```

Comment:
Here and elsewhere you've added type annotations. Does the new implicit conversion make these necessary? Is there a chance that the new conversion will break downstream code (e.g. in Sandstorm)?

Reply:
Oh sorry, please ignore the second commit in this PR. I was trying to figure out why it doesn't build on MSVC 2015, but I think I'm just going to give up on it and require MSVC 2017.